### DISCUSSION TITLE
Allow issue to have deadline/timeline

### DISCUSSION THREAD

**Discussion by chaiwa-berian** [community/community#10231] [Projects and Issues] [answered]
I would like to be able to assign a deadline to an issue

**Comment by vvdavidova** [comment, marked as answer]
What I used for that is to create a new field called 'Deadline', 'Due date' or whatever you prefer to call it, select Date as input and then assign deadlines for issues. 
I hope that helps :)

**Reply by chaiwa-berian** [reply]
Thanks, that works!
